Source/WebCore/platform/network/ResourceRequestBase.h

```diff
@@ -128,7 +128,9 @@ class ResourceRequestBase {
     WEBCORE_EXPORT void setURL(const URL& url);
 
     void redirectAsGETIfNeeded(const ResourceRequestBase &, const ResourceResponse&);
-    WEBCORE_EXPORT ResourceRequest redirectedRequest(const ResourceResponse&, bool shouldClearReferrerOnHTTPSToHTTPRedirect) const;
+
+    enum class SetHash : bool { No, Yes };
```

Comment:
Maybe this should be named `ShouldSetHash`?

Reply:
Will do